index.html

```diff
@@ -1271,12 +1271,18 @@ <h2>Verification Methods</h2>
 described in Section [[[#verification-methods]]]. When it is used in either
 place, its purpose is essentially the same; that is, it expresses one or more
 entities that are authorized to perform certain actions associated with the
-resource with which it is associated.
+resource with which it is associated. Note however that for [=verification methods=] it is only
+an assertion made by the `controller` of the [=controlled identifier document=] and this 
+assertion may be true or false. It is not necessarily a statement of fact. 
+To ensure that a [=verification method=] 
+is bound to a particular [=controller=], one must go from the expression of the [=verification method=]
+to its [=controlled-identifier-document=], and then ensure that the latter contains a reference to 
+the [=verification method=] and to its [=controller=]. 
             </p>
             <p>
 In the case of the [=controller=] of a [=controlled identifier document=], the
 [=controller=] can update the content of the document. In the case of the
-[=controller=] of a [=verification method=], the [=controller=] can generate
+[=controller=] of a [=verification method=], the actual [=controller=] can generate
 proofs that satisfy the method.
```

Comment:
I do not understand what `the actual [=controller=]` is supposed to mean, nor what it is suggesting is `the ersatz [=controller=]`.
```suggestion
[=controller=] of a [=verification method=] can generate
proofs that satisfy the method.
```

Reply:
The actual controller refers to the actual controller, rather than the asserted controller. Since the assertion can be true or false, if it is true then the asserted controller is the actual controller, if it is false, then the asserted controller is not actually the controller.

Reply:
I think I see what you mean. Perhaps this will communicate that better?
```suggestion
<em>actual</em> [=controller=] of a [=verification method=] —
which the `controller` of the [=controlled identifier document=]
may not have <em>asserted</em>
as the [=controller=] of the [=verification method=] —
can generate proofs that satisfy the method.
```

Reply:
I have suggested alternative wording to the above which I hope makes it clearer to readers.